hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -144,10 +146,13 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     String bucketKey = metadataManager.getBucketKey(volumeName, bucketName);
 
     IOException exception = null;
-    metadataManager.getLock().acquireVolumeLock(volumeName);
-    metadataManager.getLock().acquireBucketLock(volumeName, bucketName);
-    try {
+    boolean acquiredBucketLock = false;
+    metadataManager.getLock().acquireLock(VOLUME_LOCK, volumeName);
 
```

Comment:
nit: if acquireLock can return true , the  you can write acquiredBucketLock = acquireLock, just eliminates 2 lines of code

Reply:
Fixed it..